buildSrc/src/main/kotlin/uk/gov/justice/digital/hmpps/gradle/CloudPlatformTask.kt

```diff
@@ -9,7 +9,7 @@ import org.gradle.api.tasks.Internal
 import org.gradle.api.tasks.Optional
 import org.gradle.api.tasks.options.Option
 import org.gradle.api.tasks.options.OptionValues
-import org.gradle.configurationcache.extensions.serviceOf
+import org.gradle.internal.extensions.core.serviceOf
```

Comment:
using internal functions is a bit naff 🙁

Reply:
CIRCLE ERROR given is due _deprecated features_
 Test should return client error for invalid usage type() PASSED

uk.gov.justice.digital.hmpps.locationsinsideprison.resource.LocationResidentialResourceTest

  PUT /locations/{id}/**convert-cell-to-non-res-cell**

    HappyPath

      Test can update convert cell to non res cell() FAILED (11.5s)

      org.awaitility.core.ConditionTimeoutException: Lambda expression in org.awaitility.kotlin.AwaitilityKt expected the predicate to return <true> but it returned <false> for input of <true> within 10 seconds.
          at app//uk.gov.justice.digital.hmpps.locationsinsideprison.resource.LocationResidentialResourceTest$ConvertCellToNonResCell$HappyPath.can update convert cell to non res cell(LocationResidentialResourceTest.kt:1417)
      Caused by: java.util.concurrent.TimeoutException
          at uk.gov.justice.digital.hmpps.locationsinsideprison.resource.LocationResidentialResourceTest$ConvertCellToNonResCell$HappyPath.can update convert cell to non res cell(LocationResidentialResourceTest.kt:1417)